src/reports/package/reporter-factory.ts

```diff
@@ -130,9 +128,38 @@ const summaryResultsReportGenerator = (parameters: SummaryReportParameters) => {
     return new SummaryResultsReport(deps, parameters, toolData);
 };
 
+const combinedResultsReportGenerator = (parameters: CombinedReportParameters) => {
+    const { serviceName, axeVersion, userAgent } = parameters;
+
+    const toolData = createToolData(
+        serviceName,
+        '',
```

Comment:
This feels a little off; can we pass a serviceVersion param along with the serviceName, rather than leaving this blank? Or if we do want to omit it, can we make it more clear in the typings (ie, allow `createToolData` to accept `null` explicitly, rather than using an empty string)?

(I do see that you're just matching what the other summary report is already doing, so feel free to defer this comment from the PR if you'd rather consider it a separate change)